internal/indexer/definitions/digitalcore.yaml

```diff
@@ -0,0 +1,61 @@
+---
+#id: digitalcore
+name: DigitalCore
+identifier: digitalcore
+description: DigitalCore (DC) is a private torrent tracker for General / 0 Day.
+language: en-us
+urls:
+  - https://digitalcore.club
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: gazelle
+settings:
+  - name: passkey
+    type: secret
+    label: PassKey
+    help: Settings -> Security -> Passkey.
+
+irc:
+  network: DigitalCore
+  server: irc.digitalcore.club
+  port: 7000
+  tls: true
+  channels:
+    - "#announce"
+  announcers:
+    - ENDOR
+  settings:
+    - name: nickserv.account
+      type: text
+      required: false
```

Comment:
Change to `true` to require a username, even if it's not necessarily a nickserv account.